src/langcheck/metrics/ja/reference_free_text_quality.py

```diff
@@ -8,44 +8,37 @@
 from langcheck.metrics._validation import (validate_parameters_answer_relevance,
                                            validate_parameters_reference_free)
 from langcheck.metrics.en._openai import OpenAIBasedEvaluator
+from langcheck.metrics.eval_clients import EvalClient
 from langcheck.metrics.metric_value import MetricValue
 from langcheck.metrics.scorer.hf_models import \
     AutoModelForSequenceClassificationScorer
 from langcheck.utils.progess_bar import tqdm_wrapper
 
+from ..prompts._utils import get_template
 
-def sentiment(generated_outputs: List[str] | str,
-              prompts: Optional[List[str] | str] = None,
-              model_type: str = 'local',
-              openai_client: Optional[OpenAI] = None,
-              openai_args: Optional[Dict[str, str]] = None,
-              local_overflow_strategy: str = 'truncate',
-              *,
-              use_async: bool = False) -> MetricValue[Optional[float]]:
+
+def sentiment(
+        generated_outputs: List[str] | str,
+        prompts: Optional[List[str] | str] = None,
+        model_type: str = 'local',
+        llm_client: Optional[EvalClient] = None,
```

Comment:
Instead of having both of these, maybe we can simplify this down to
```suggestion
        eval_client: str | EvalClient = 'local',
```

Reply:
Oh yeah looks good!

I renamed the argument to `eval_model` to make it clear that the argument doesn't only take `EvalClient` here.